soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -1057,6 +1289,227 @@ void DrawQuestStatusTab() {
     ImGui::PopItemWidth();
 }
 
+void DrawPlayerTab() {
+    if (gGlobalCtx) {
+        Player* player = GET_PLAYER(gGlobalCtx);
+        std::string curSword, curShield, curTunic, curBoots;
```

Comment:
Like the minigames above these should be `const char*`.
Also they should be on their own lines per OOT's formatting.